documentation/src/main/asciidoc/topics/proc_cli_xsite_sites.adoc

```diff
@@ -0,0 +1,25 @@
+[id='manage_sites-{context}']
+= Bringing Sites Offline and Online
+Take sites offline manually and bring sites back online.
+
+.Procedure
+
+* Check if sites are online or offline with the `site status` command:
++
+----
+//containers/default]> site status -s LON
+----
+
+* Bring sites online with the `bring-online` command:
++
+----
+//containers/default]> site bring-online --cache=customers --site=NYC
+----
+
+* Take sites offline with the `take-offline` command:
++
+----
+//containers/default]> site take-offline --cache=customers --site=NYC
+----
+
+For more information and examples, run the `help site` command.
```

Comment:
the help message (returned by `help site`) needs to be updated as well.

Reply:
I've updated the help resources for the site command.